>I don't see any related code.

They are already uploaded, I just changed the suffix to use the contents of `VERSION` instead of timestamp + commit, this way `status-react` can download it using it's contents of `STATUS_GO_VERSION` file.

Also, this PR is not done because it fails due to a hardcoded name:
```log
+ make prepare-release
rm -rf /tmp/release-0.17.1-beta
mkdir -p /tmp/release-0.17.1-beta
mv build/bin/statusgo-android-16.aar /tmp/release-0.17.1-beta/status-go-android.aar
mv: cannot stat 'build/bin/statusgo-android-16.aar': No such file or directory
Makefile:219: recipe for target 'prepare-release' failed
make: *** [prepare-release] Error 1
```
https://ci.status.im/job/status-go/job/parallel/121/console

<blockquote><img src="/static/e7749a62/favicon.ico" width="48" align="right"><div><strong><a href="https://ci.status.im/job/status-go/job/parallel/121/console">status-go » parallel #121 Console [Jenkins]</a></strong></div></blockquote>

--- a/_assets/ci/Jenkinsfile
+++ b/_assets/ci/Jenkinsfile
@@ -3,8 +3,8 @@ pipeline {
 
   parameters {
     booleanParam(
-      name: 'GITHUB_RELEASE',
-      description: 'Enable this to create a new GitHub release.',
+      name: 'RELEASE',
+      description: 'Enable to create a new release on GitHub and DigitalOcean Space.',
       defaultValue: false,
     )
   }
@@ -50,13 +50,16 @@ pipeline {
             } } }
             stage('Archive') { steps {
               sh """
-                mv ${env.STATUS_PATH}/build/bin/statusgo-android-16.aar \
+                cp ${env.STATUS_PATH}/build/bin/statusgo-android-16.aar \
                   ${env.WORKSPACE}/status-go-android-${lib.suffix()}.aar
               """
               archiveArtifacts("status-go-android-${lib.suffix()}.aar")
             } }
             stage('Upload') { steps { script {
               lib.uploadArtifact("status-go-android-${lib.suffix()}.aar")
+            } } }
+            stage('Cleanup') { steps { script {
+              sh "rm -f ${env.WORKSPACE}/*.aar"
             } } }
           }
         }
@@ -70,7 +73,7 @@ pipeline {
             } } }
             stage('Archive') { steps {
               sh """
-                mv ${env.STATUS_PATH}/build/bin/statusgo-ios-9.3-framework/status-go-ios.zip \
+                cp ${env.STATUS_PATH}/build/bin/statusgo-ios-9.3-framework/status-go-ios.zip \
                   ${env.WORKSPACE}/status-go-ios-${lib.suffix()}.zip
               """
               archiveArtifacts("status-go-ios-${lib.suffix()}.zip")
@@ -78,22 +81,27 @@ pipeline {
             stage('Upload') { steps { script {
               lib.uploadArtifact("status-go-ios-${lib.suffix()}.zip")
             } } }
+            stage('Cleanup') { steps { script {
+              sh "rm -f ${env.WORKSPACE}/*.zip"
+            } } }
           }
         }
       }
     }
-    stage('Release') { when { expression { params.GITHUB_RELEASE == true } }
+    stage('Release') { when { expression { params.RELEASE == true } }
       steps {
-        sh 'make prepare-release'
-        withCredentials([[
-          $class: 'UsernamePasswordMultiBinding',
-          credentialsId: 'status-im-auto',
-          usernameVariable: 'GITHUB_USER',
-          passwordVariable: 'GITHUB_TOKEN'
-        ]]) {
-          sh "yes | make release release_branch=${gitBranch}"
+        dir(env.STATUS_PATH) {
+          sh 'make prepare-release'
+          withCredentials([[
+            $class: 'UsernamePasswordMultiBinding',
+            credentialsId: 'status-im-auto',
+            usernameVariable: 'GITHUB_USER',
+            passwordVariable: 'GITHUB_TOKEN'
+          ]]) {
+            sh "yes | make release release_branch=${lib.gitBranch()}"
+          }
+          sh 'make clean-release'
         }
-        sh 'make clean-release'
       }
     }
   }

--- a/_assets/ci/lib.groovy
+++ b/_assets/ci/lib.groovy
@@ -8,7 +8,11 @@ def timestamp() {
 }
 
 def suffix() {
-  return "${timestamp()}-${gitCommit()}"
+  if (params.RELEASE == true) {
+    return 'v' + readFile("${env.WORKSPACE}/${env.STATUS_PATH}/VERSION").trim()
+  } else {
+    return "${timestamp()}-${gitCommit()}"
+  }
 }
 
 def gitBranch() {

